main/linux-vanilla/APKBUILD

```diff
@@ -7,7 +7,7 @@ case $pkgver in
 	*.*.*)	_kernver=${pkgver%.*};;
 	*.*) _kernver=$pkgver;;
 esac
-pkgrel=0
+pkgrel=1
```

Comment:
This would of course necessitate a bump of other modules, but it'd be best if we could just apply the configuration differences with the next kernel bump